src/js/page.js

```diff
@@ -81,14 +81,22 @@ export default {
      * @param {jQuery} $e - Element on page to flip to
      */
     flipToPageContaining( $e ) {
+        const e = $e[ 0 ];
         let $closest;
-
-        $closest = $e.closest( '[role="page"]' );
-        $closest = ( $closest.length === 0 ) ? $e.find( '[role="page"]' ) : $closest;
-
-        // If $e is a comment question, and it is not inside a group, there may be no $closest.
-        if ( $closest.length ) {
-            this._flipTo( $closest[ 0 ] );
+        $closest = e.closest( '[role="page"]' );
```

Comment:
We are using the convention to define jQuery objects with a $ prefix. This is not a jQuery object anymore, so it should be `closest` to avoid confusion.

Reply:
code updated